dev/ci/user-overlays/13321-ppedrot-mv-evaluable-global-ref-out-of-kernel.sh

```diff
@@ -0,0 +1,5 @@
+if [ "$CI_PULL_REQUEST" = "13321" ] || [ "$CI_BRANCH" = "mv-evaluable-global-ref-out-of-kernel" ]; then
+
+    overlay equations https://github.com/ppedrot/Coq-Equations mv-evaluable-global-ref-out-of-kernel
+
+fi
```

Comment:
You need to add the PR number as last argument of `overlay` (I got trapped too) and remove the now useless `if`:
```suggestion
overlay equations https://github.com/ppedrot/Coq-Equations mv-evaluable-global-ref-out-of-kernel 13321
```

Reply:
This was generated by the `dev/tools`  script IIRC, it probably means it's currently buggy. cc @SkySkimmer

Reply:
Anyways CI now compiles properly.